ruby_event_store-browser/public/ruby_event_store_browser.css

```diff
@@ -0,0 +1,2 @@
pe:none;padding:0}dl,dt,ol,ul{margin:0}dt{font-weight:600}dd,figure{margin:0}img,picture{margin:0;max-width:100%}table{border-collapse:collapse;margin:1.5em 0;table-layout:fixed;text-align:left;width:100%}thead{line-height:1.2;vertical-align:bottom}tbody{vertical-align:top}th{border-bottom:1px solid #ced4da;padding:0 0 .75rem;text-transform:uppercase;font-size:1.2rem;color:#adb5bd}td{padding:.75rem 0 0}html{color:#343a40;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:62.5%;line-height:1.6;-webkit-font-smoothing:antialiased}body{font-size:1.6rem}h1,h2,h3,h4,h5,h6{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:2em;line-height:1.2;margin:0 0 .75em}p{margin:0 0 .75em}a{color:#cf423b;text-decoration-skip:ink;transition:color .15s ease}a:hover{color:shade(#cf423b,25%)}a:focus{outline:3px solid rgba(207,66,59,.6);outline-offset:2px}hr{border:0;border-bottom:1px solid #ced4da;margin:1.5em 0}.frame{min-height:100vh;width:100%;background-color:#f8f9fa}.frame__body{background-color:#fff}.navigation{height:6rem;background-color:#cf423b}.navigation:after{clear:both;content:"";display:block}.navigation__brand{width:calc(33.33333% - 26.66667px);justify-content:flex-start}.navigation__brand,.navigation__links{float:left;margin-left:20px;display:flex;height:100%;align-items:center}.navigation__links{width:calc(66.66667% - 33.33333px);justify-content:flex-end}.navigation__link{margin-left:1rem;padding:.5rem;text-decoration:none;font-weight:700;font-size:1.4rem;color:#f1f3f5}.navigation__link--active,.navigation__link:hover{border-radius:4px;background-color:#ad302a;color:#f8f9fa}.navigation__logo{text-decoration:none;font-weight:600;color:#f8f9fa}.footer{border-top:1px solid #ced4da;padding:1rem 0}.footer:after{clear:both;content:"";display:block}.footer__links{width:calc(100% - 40px);float:left;margin-left:20px;display:flex;justify-content:center;font-size:1.4rem;color:#adb5bd}.footer__link{margin:0 0 0 1rem;color:#adb5bd}.footer__link:before{content:"•";font-weight:700;margin:0 1rem 0 0;color:#dee2e6;display:inline-block}.browser{padding:2rem 0}.browser:after{clear:both;content:"";display:block}.browser__title{font-size:2.2rem}.browser__pagination,.browser__results,.browser__title{width:calc(100% - 40px);float:left;margin-left:20px}.event{padding:2rem 0}.event:after{clear:both;content:"";display:block}.event__title{font-size:2.2rem}.event__body,.event__title{width:calc(100% - 40px);float:left;margin-left:20px}.pagination{display:flex;margin-bottom:.75em}.pagination__page{height:3.6rem;background:none;text-align:center;padding:1rem;margin-right:.5rem;color:#cf423b;border:1px solid #cf423b;font-weight:400;font-size:90%}.pagination__page:focus{outline:none}.results__link{text-decoration:none}.results__empty{display:flex;align-items:center;justify-content:center;min-height:20rem}.u-align-right{text-align:right}
```

Comment:
What is the reason to commit ignored css file?

Reply:
That's production one, it's not ignored. Development ones are ignored.